backend/lobby/consumers.py

```diff
@@ -12,11 +12,20 @@
 from game.player.player import Player
 from game.player.player_status_resolver import player_status_summary_to_JSON
 from game.values.constants import DEFAULT_DICE_TO_ROLL, DEFAULT_RE_ROLL_COUNT
+from game.values.exceptions import InsufficientFundsException
 from lobby.consumers_common import save_game, reconstruct_game, create_send_response_to_client
 from lobby.server_message_types import PLAYER_STATUS_UPDATE_RESPONSE, BEGIN_TURN_RESPONSE, SERVER_RESPONSE, \
     DICE_ROLLS_RESPONSE, CARD_STORE_RESPONSE
 
 
+def dice_vals_log_message(player_name, values):
+    msg = "{} rolled: ".format(player_name)
+    for val in values[:-1]:
+        msg += val.name + ", "
+    msg += values[-1].name
```

Comment:
This can be rewritten to be simpler/1 line

`msg = "%s rolled: %s" % (player_name, ", ".join([val.name for val in values]))`

you might have to wrap the val.name in str() though